src/Uno.UI/System/DispatcherQueueTimer.cs

```diff
@@ -0,0 +1,147 @@
+using System;
+using System.Threading;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.Foundation;
+
+namespace Microsoft.System
+{
+	public partial class 
+		DispatcherQueueTimer 
+	{
+		private static class States
+		{
+			public const int Idle = 0;
+			public const int Running = 1;
+		}
+
+		private int _state = States.Idle;
+		private TimeSpan _interval;
+		private DateTimeOffset _lastTick;
+
+		public event TypedEventHandler<DispatcherQueueTimer, object> Tick;
+
+		public TimeSpan Interval
+		{
+			get => _interval;
+			set
+			{
+				if (value < TimeSpan.Zero)
+				{
+					throw new ArgumentException(nameof(value)); // WinUI throws a ArgumentException, not and ArgumentOutOfRangeException
+				}
+
+				if (_interval != value)
+				{
+					_interval = value;
+					if (IsRunning)
+					{
+						Restart(value);
+					}
+				}
+			}
+		}
+
+		public bool IsRunning => _state == States.Running;
+
+		public bool IsRepeating
+		{
+			get => true;
+			[global::Uno.NotImplemented]
+			set => global::Windows.Foundation.Metadata.ApiInformation.TryRaiseNotImplemented("Windows.System.DispatcherQueueTimer", "bool DispatcherQueueTimer.IsRepeating");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the 'value' parameter in this property set accessor declaration.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5475653)